docs/advanced-ai/examples/understand-chains.md

```diff
@@ -0,0 +1,20 @@
+---
+title: What is a chain in AI?
+description: Understand chains in the context of AI. Learn about chains in n8n.
+contentType: explanation
+---
+
+# What is a chain in AI?
+
+Chains bring together different components of AI to create a cohesive system. They set up a sequence of calls between the components. These components can include models and memory (though note that in n8n chains can't use memory).
+
+
+## Chains in n8n
+
+n8n provides three chain nodes:
+
+* [Basic LLM Chain](/integrations/builtin/cluster-nodes/root-nodes/n8n-nodes-langchain.chainllm/): use to interact with an LLM, without any additional components.
+* [Retrieval Q&A Chain](/integrations/builtin/cluster-nodes/root-nodes/n8n-nodes-langchain.chainretrievalqa/): can connect to a vector store using a retriever. Use this if you want to create a workflow that supports asking questions about specific documents.
```

Comment:
Might be worth mentioning here that you can also connect Workflow Retriever to fetch context using workflow